config.py

```diff
@@ -817,3 +826,7 @@ def create_transaction(db):
 
     FEATURE_RESTRICTED_USERS = False
     RESTRICTED_USERS_WHITELIST: Optional[List[str]] = None
+
+    QUOTA_INVALIDATE_TOTALS = True
+    RESET_CHILD_MANIFEST_EXPIRATION = True
+    PERMANENTLY_DELETE_TAGS = True
```

Comment:
They should be enabled out of the box since they're required for Quota to work properly. They should only be able to be disabled in certain scenarios. These options are also all internal only properties.